test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsTests.cs

```diff
@@ -2043,35 +2043,80 @@ public void LoadSettings_AddsV3ToEmptyConfigFile_OnlyFirstTime()
 
                 // Assert
                 var text = SettingsTestUtils.RemoveWhitespace(File.ReadAllText(Path.Combine(mockBaseDirectory, "TestingGlobalPath", "NuGet.Config")));
+                var v3feed = SettingsTestUtils.RemoveWhitespace(@"<add key=""nuget.org"" value=""https://api.nuget.org/v3/index.json"" protocolVersion=""3"" />");
                 var result = SettingsTestUtils.RemoveWhitespace(@"<?xml version=""1.0"" encoding=""utf-8""?>
 <configuration>
-    <packageSources>
-        <add key=""nuget.org"" value=""https://api.nuget.org/v3/index.json"" protocolVersion=""3"" />
-    </packageSources>
 </configuration>");
 
+                text.Should().NotContain(v3feed);
                 text.Should().Be(result);
```

Comment:
`text.Should().Be(result)` looks like it already asserts that it does not contain nuget.org. The `.NotContains` looks like it's redundant.

Reply:
Thanks for reviewing @zivkan !
I'm not sure if my understanding is right:
There are two levels, if the V3feed is added or not, if other parts are correct. 
So if the first assertion passes, but the second assertion failed, we will know it's something other than V3feed is wrong.

If it's not worth adding that, I'll remove it.

Reply:
A very simplified example is:
```cs
var text = "AC";
var result = "AC";
result.Should().NotContain("B");
result.Should().Be(text);
```

Since the last assertion is effectively `text == result`, and `text` was hardcoded in the test setup, we know it doesn't contain "B". The `result.Should().NotContain("B")` assertion doesn't validate anything that isn't already validated.

This also made me realize that the variable names are confusing (I know you modified, then copied an existing test). The variable `result` makes it sound like it's the outcome of the test's "act" step, and `text` sounds like it should be a hardcoded string.  I really like xunit's `actual` and `expected` parameter names, because it's really clear what they should be. Anyway, minor nitpick about the naming. The main point is that the `NotContain` assertion will not catch any regressions. The `Be` assertion already catches the same scenario.

Reply:
Thanks for the detailed explanation. Removed redundant assertions and used actual and expected names.